easybuild/easyconfigs/g/gcsfs/gcsfs-2023.12.2-foss-2023a.post1.eb

```
@@ -0,0 +1,91 @@
+easyblock = "PythonBundle"
+
+name = 'gcsfs'
+version = '2023.12.2'
+versionsuffix = '.post1'
```

@pavelToman the `.post1` is just part of the version, so:

```suggestion
version = '2023.12.2.post1'
```

Do note that this implies renaming the easyconfig file as well